<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentdashboard</class>
 <widget class="QDialog" name="studentdashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTextBrowser" name="toptext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#354680;&quot;&gt;Hello STUDENT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>961</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="RequestTab">
    <property name="styleSheet">
     <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
    </property>
    <attribute name="title">
     <string>Request</string>
    </attribute>
    <widget class="QTextBrowser" name="title">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>431</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#354680;&quot;&gt;Create a new Request üê≥&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="formGroupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>431</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="sub_label">
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="subjects">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="urg_label">
        <property name="text">
         <string>Urgency:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="urgency">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="desc_label">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="desc">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Description</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="req_status">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>390</y>
       <width>431</width>
       <height>65</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="createreq_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>431</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Create Request</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SearchTab">
    <property name="styleSheet">
     <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
    </property>
    <attribute name="title">
     <string>Search</string>
    </attribute>
    <widget class="QFrame" name="search_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>901</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextBrowser" name="search_out"/>
      </item>
      <item>
       <widget class="QGroupBox" name="searchbar">
        <layout class="QHBoxLayout" name="search">
         <item>
          <widget class="QComboBox" name="searchoptions">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="search_in">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchtutor_btn">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Search Tutor</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="SessionsTab">
    <property name="styleSheet">
     <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
    </property>
    <attribute name="title">
     <string>Sessions</string>
    </attribute>
    <widget class="QFrame" name="session">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>901</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTextBrowser" name="sessions_out"/>
      </item>
      <item>
       <widget class="QGroupBox" name="inputbar">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="rating_in">
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tutor_user">
           <property name="placeholderText">
            <string>Tutor Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="comments">
           <property name="placeholderText">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ratetutor_btn">
           <property name="text">
            <string>Rate Tutor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="session_in">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="session_id">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
              <property name="placeholderText">
               <string>Session ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="endsession_btn">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>End Session</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="refresh_btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
